### ISSUE TITLE
How to get to predicted label-data in video example on UCF11 dataset? (python API)

### ISSUE THREAD

**Issue by nSeck** [Microsoft/CNTK#1572]
Hey there,

I'm still working on an image regression problem with video data. Following question occured while working on this [video example](https://github.com/Microsoft/CNTK/tree/master/Examples/Video/GettingStarted) of yours:

How do I access the predicted label_data from trainer.test_minibatch() in
  
```
    while test_reader.has_more():
        videos, labels, current_minibatch = test_reader.next_minibatch(minibatch_size)
        # minibatch data to be trained with
        metric_numer += trainer.test_minibatch({input_var : videos, label_var : labels}) * current_minibatch
        metric_denom += current_minibatch
        # Keep track of the number of samples processed so far.
        minibatch_index += 1
```
In Brainscript there is the "write action" to do this and in [FastRCNN-Example](https://github.com/Microsoft/CNTK/blob/master/Examples/Image/Detection/FastRCNN/A2_RunCntk_py3.py) following seems to work:
```
   print("Evaluating Fast R-CNN model for %s images." % num_test_images)
    results_file_path = base_path + "test.z"
    with open(results_file_path, 'wb') as results_file:
        for i in range(0, num_test_images):
            data = test_minibatch_source.next_minibatch(1, input_map=input_map)
            output = model.eval(data)
            out_values = output[0, 0].flatten()
            np.savetxt(results_file, out_values[np.newaxis], fmt="%.6f")
            if (i+1) % 100 == 0:
print("Evaluated %s images.." % (i+1))
```
In the given video example this does not work, because the test_reader.next_minibatch()-function is self defined and does not have the same output.

Any help would be appreciated.
Thanks for your hard work on CNTK!

**Comment by nSeck** [comment]
Found the solution myself. The data has to be structured like:

`data = {input_var : videos, label_var : labels}`

and then you get the output via

`output = z.eval(data)`

The problem I was getting, was because of loading a pre-trained model.
